### PR TITLE
first stab at 2018 website

### PR DESCRIPTION
**Pull request by tyarkoni**
This isn't quite complete yet, but assuming you're happy with it, it's probably good enough to publicize. Missing items include better faculty bios and more FAQs. Otherwise I think we're in pretty good shape. But see what you think, @arokem, and I'm happy to make changes.

### CODE REVIEW

File: _includes/faq.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+			<h5>I'm a [faculty member/research assistant/postdoc/1st year graduate student/...]; can I still apply?</h5>
+			<p class="a">Applicants at all career stages are welcome to apply to Neurohackademy. In the past, we've admitted participants who range in experience from (pre-graduate school) research assistant all the way through tenured full professor. We don't consider career stage as an explicit factor in the evaluation process. That said, the admission criteria do <i>implicitly</i> give some preference to graduate students and postdocs, as applicants at these career stages are most likely to benefit from the course (undergrads and research assistant are less likely to have had time to develop the necessary basic computational and/or neuroscience skills, whereas current faculty typically have less opportunity to directly apply any skills they might gain). Our general adeapplication process shouldn't take more than an hour or two of your time, and if you don't get in this year, you're always welcome to apply again next year!</p>
```
Comment (arokem) [inline _includes/faq.html:24]: Typo: "adeapplication"

File: _includes/faq.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+			<h5>I'm an early-career researcher with a strong background in neuroscience, but my computational skills aren't very strong. Can I attend?</h5>
+
+			<p class="a">The primary goal of Neurohackademy is to help researchers develop the computational skills they need in order to make the most of large, rich neuroscience datasets. So we certainly don't expect applicants to come in with extensive prior programming experience (and if anything, that might hurt one's chances of admission, since there's no point in us trying to teaching someone things they already know!). That said, we do assume that all participants have a minimal level of programming competency--without which a participant will probably find it much more difficult to benefit from the course.</p>
```
Comment (arokem) [inline _includes/faq.html:18]: I'd remove the parenthetical "(and if anything..." -- it just opens up more questions, in my opinion

**Review by arokem — COMMENTED** [review]
Looks great. I had a couple of small comments, but I think that we can fix after merging, so I am going to go ahead and merge.

**Comment by tyarkoni** [comment]
Can you make me a collaborator? I don't mind opening PRs if you prefer to do it that way, and I'll still go that route for major changes, but it seems like overkill for minor changes like the ones you flag.

**Comment by arokem** [comment]
Ha! You're not a collaborator here? That's silly.

**Comment by arokem** [comment]
You are now also an owner of the org

**Comment by tyarkoni** [comment]
Woohoo!

As my first order of business, I hereby decree that all commit messages and thread comments in any repo under this org must abruptly end without

**Pull request merged**